rbac/management/group/view.py

```diff
@@ -32,6 +32,7 @@
 from management.permissions import GroupAccessPermission
 from management.principal.model import Principal
 from management.principal.proxy import PrincipalProxy
+from management.principal.serializer import PrincpalSerializer
```

Comment:
Minor type here and below where it's called. 
```suggestion
from management.principal.serializer import PrincipalSerializer
```